src/xhr.js

```diff
@@ -19,6 +19,11 @@ export function xhrImpl(url, { method, headers, body, onNext, onError, onComplet
     const patchResolver = new PatchResolver({ onResponse: r => onNext(r) });
 
     function onReadyStateChange() {
+        // The request failed, do nothing and let the error event fire
+        if (this.readyState === this.DONE && this.status === 0) {
```

Comment:
It was trying to parse the xhr response below and was throwing an error. Figured it's just better to bail out early.